src/components/LogViewer/LogViewer.tsx

```diff
@@ -105,24 +112,28 @@ export const LogViewer = ({ data }: ILogViewerProps) => {
           <Button
             onClick={() => {
               setIsPaused(false);
+              logViewerRef.current?.scrollToItem(lineCount);
```

Comment:
Why this change is needed?

Reply:
Disabling the pause state works only for the dynamic log variant. The static log variant needs 'scrolling to item' .

Reply:
done